src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
+$block-size: 100px;
+$block-color: #eee;
+$calendar-padding: 10px;
+
 body {
   margin: 0;
 }
+
+@for $i from 1 to 32 {
```

Comment:
move this loop inside day selector

Reply:
not fixed
<img width="303" alt="image" src="https://user-images.githubusercontent.com/53611910/208255870-dcd91c35-26fa-4303-9bf4-02ede4d83732.png">